### PR TITLE
Add abandoned cart example to ecommerce demo

### PR DESCRIPTION
**Pull request by bobbyiliev**
Current eCommerce database structure:

image

Updated eCommerce database structure:

image

### CODE REVIEW

**Review by dehume — COMMENTED** [review]
Do we also need to update the [mysql_bootstrap](https://github.com/MaterializeInc/demos/blob/main/ecommerce-confluent/mysql/mysql_bootstrap.sql#L33) to add the `status` column to the table create statement?

**Comment by bobbyiliev** [comment]
> Do we also need to update the [mysql_bootstrap](https://github.com/MaterializeInc/demos/blob/main/ecommerce-confluent/mysql/mysql_bootstrap.sql#L33) to add the `status` column to the table create statement?

It seems like it's already there on [line 38](https://github.com/MaterializeInc/demos/blob/main/ecommerce-confluent/mysql/mysql_bootstrap.sql#L38)

File: ecommerce-confluent/loadgen/generate_load.py [hunk 1]
```diff
@@ -93,6 +93,7 @@ def generatePageview(viewer_id, target_id, page_type):
                 purchase_item = random.choice(item_prices)
                 purchase_user = random.randint(0, userSeedCount - 1)
                 purchase_quantity = random.randint(1, 5)
+                purchase_status = random.randint(0,1)
```
[the remark this answers is not in the document]
Comment (bobbyiliev) [reply ecommerce-confluent/loadgen/generate_load.py:96]: Good point, what would be a good ratio for this?

File: ecommerce-confluent/README.md [hunk 2]
```diff
@@ -387,6 +387,72 @@ CREATE SINK high_value_users_sink
 
 Now if you go to the [Confluent Cloud UI](https://confluent.cloud/) and navigate to the `high-value-users-sink` topic, you should see data streaming in.
 
+## Abandoned cart example
+
+We can start simple with a materialized view that aggregates all users that have not completed a purchase:
+
+```sql
+CREATE MATERIALIZED VIEW incomplate_purchases AS
```
[the remark this answers is not in the document]
Comment (bobbyiliev) [reply ecommerce-confluent/README.md:395]: Open to suggestions! What would be a good name for this instead?

File: ecommerce-confluent/loadgen/generate_load.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (dehume) [reply ecommerce-confluent/loadgen/generate_load.py:96]: No strong thoughts. 5-10%. I can't remember our exact ratio at Drizly but abandoned carts were in the minority

File: ecommerce-confluent/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (dehume) [reply ecommerce-confluent/README.md:395]: I think the name is fine. Just wasn't sure if "incomplate" was a typo

[the remark this answers is not in the document]
Comment (bobbyiliev) [reply ecommerce-confluent/README.md:395]: My dyslexia kicked in there 😆  thank you for pointing that out 🙏

**Review by dehume — APPROVED** [review]

**Review by morsapaes — APPROVED** [review]

**Pull request merged**